src/client/containers/ConfigurationCreateContainer/hooks/useConfigurationCreate.ts

```diff
@@ -0,0 +1,125 @@
+import { useEffect } from "react";
+import { useMutation } from "react-query";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+import { requester } from "../../../lib";
+import { Resource, ResourceType, WebhookEventType } from "../../../constants";
+import { ClientError } from "../../../ClientError";
+
+interface WebhookSettings {
+    webhookUrl: string;
+}
+
+const errorToText = (error: Error): string => {
+    switch (error instanceof ClientError ? error.message : "") {
+        case "User is not a member of the project":
+            return "Only project members can setup integrations settings.";
+        case "User is not a member of the styleguide":
+            return "Only styleguide members can setup integrations settings.";
+        case "Project not found":
+        case "Project is archived":
+            return "Project is not available anymore. You can select different Project/Styleguide to setup configuration.";
+        case "Styleguide not found":
+        case "Styleguide is archived":
+            return "Styleguide is not available anymore. You can select different Project/Styleguide to setup configuration.";
+        case "Only organization editor (or higher) can access project webhooks":
+        case "Only organization editor (or higher) can access styleguide webhooks":
+            return "Only organization editor (or higher) can setup integration settings.";
+        case "Only owner of the project can access webhooks":
+            return "Only owner of the project can setup integration settings.";
+        case "Only owner of the styleguide can access webhooks":
+            return "Only owner of the styleguide can setup integration settings.";
+        case "Cannot create a webhook with active status because provided URL is unhealthy":
+        case "Owner of the webhook not found. Please contact to support@zeplin.io":
+        default:
+            return "We're experiencing an issue here. Please try it later or let us know: support@zeplin.io.";
+    }
```

Comment:
```suggestion
    switch (error instanceof ClientError ? error.message : "") {
        case "User is not a member of the project":
            return "Only project members can setup integrations.";
        case "User is not a member of the styleguide":
            return "Only styleguide members can setup integrations.";
        case "Project not found":
        case "Project is archived":
            return "Project is not available anymore. You can select different Project/Styleguide to setup configuration.";
        case "Styleguide not found":
        case "Styleguide is archived":
            return "Styleguide is not available anymore. You can select different Project/Styleguide to setup configuration.";
        case "Only organization editor (or higher) can access project webhooks":
        case "Only organization editor (or higher) can access styleguide webhooks":
            return "Only organization editor (or higher) can setup integrations.";
        case "Only owner of the project can access webhooks":
            return "Only owner of the project can setup integrations.";
        case "Only owner of the styleguide can access webhooks":
            return "Only owner of the styleguide can setup integrations.";
        case "Cannot create a webhook with active status because provided URL is unhealthy":
        case "Owner of the webhook not found. Please contact to support@zeplin.io":
        default:
            return "We're experiencing an issue here. Please try it later or let us know: support@zeplin.io.";
    }
```